examples/with_winit/README.md

```diff
@@ -0,0 +1,25 @@
+## Usage
+
+Running the viewer without any arguments will render a built-in set of public-domain Lottie images:
```

Comment:
I don't think the (one?) built in file is public domain in the legal sense

Reply:
fixed